### PR TITLE
CSIP5: Create testCase.xml

### PR DESCRIPTION
**Pull request by koit**

### CODE REVIEW

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml [hunk 1]
```diff
@@ -1,23 +1,88 @@
-<!-- Root element for an individual test case, allows these to be wrapped into
-XML lists -->
-<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
-  <!-- Unique ID for the test case -->
-  <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP5"/>
-  <!-- URL references to requirements for convenient lookup -->
+<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd" testable="TRUE">
+  <id specification="CSIP" version="2.0.3" requirementId="CSIP5"/>
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/mets-root/CSIP5/testCase.xml]: 2.0.4 is the latest

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml [hunk 2]
```diff
@@ -1,23 +1,88 @@
...
-  <!-- List of requirments that this test case depends on in addition to the
-  main requirememt, e.g. general requirments on the form of IDs -->
+  <requirementText>
+      <name>Other Content Information Type Specification</name>
+      <location>mets[@csip:CONTENTINFORMATIONTYPE='OTHER']/@csip:OTHERCONTENTINFORMATIONTYPE</location>
+    <description>
+      When the mets/@csip:CONTENTINFORMATIONTYPE has the value "OTHER" the mets/@csip:OTHERCONTENTINFORMATIONTYPE must state the content information type.
+    </description>
+    <cardinality>0..1</cardinality>
+    <!--  It is actually a MUST or more precisely, a "conditional MUST":
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/mets-root/CSIP5/testCase.xml:13]: Yes - I think we need to discuss this with A2. How to represent conditional MUSTs in all of our specifications?

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml [hunk 3]
```diff
@@ -1,23 +1,88 @@
...
+          <path></path>
+          <description>
+            Minimal IP that matches all MUST requirements, mets/@csip:CONTENTINFORMATIONTYPE = "OTHER", mets/@csip:OTHERCONTENTINFORMATIONTYPE = "SIARDUK".
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <!-- This rule is implicit in the specification and is important for CSIP4 and CSIP5 to work as intended. -->
+    <rule id="2">
+      <description>If mets/@csip:OTHERCONTENTINFORMATIONTYPE exists then mets/@csip:CONTENTINFORMATIONTYPE MUST be "OTHER".</description>
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/mets-root/CSIP5/testCase.xml:112]: Or maybe also "MIXED"?

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml [hunk 4]
```diff
@@ -1,23 +1,88 @@
...
+      <error level="ERROR">
+        <message>
+          mets/@csip:OTHERCONTENTINFORMATIONTYPE exists although mets/@csip:CONTENTINFORMATIONTYPE is not "OTHER". MUST be: mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST only exist if mets/@csip:CONTENTINFORMATIONTYPE = "OTHER".
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/mets-root/CSIP5/testCase.xml:115]: Or maybe also "MIXED"?

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
I think there might lack a test to make sure that the value in @othercontentinformationtype is not empty.
Maybe it can come with the list of the values that must not exist in ruleID=1:
- vocabulary items
- empty string

**Comment by koit** [comment]
> I think there might lack a test to make sure that the value in @othercontentinformationtype is not empty.
> Maybe it can come with the list of the values that must not exist in ruleID=1:
> 
> * vocabulary items
> * empty string

Fixed. The rules for emptiness existed but were part of CSIP4 testCase - I've now moved them here.

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml (hunk 1 above)
Comment (koit) [reply corpora/csip/metadata/mets-root/CSIP5/testCase.xml]: Fixed.

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml (hunk 2 above)
Comment (koit) [reply corpora/csip/metadata/mets-root/CSIP5/testCase.xml:13]: I've created a ticket about it: https://github.com/DILCISBoard/E-ARK-CSIP/issues/657.

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml (hunk 3 above)
Comment (koit) [reply corpora/csip/metadata/mets-root/CSIP5/testCase.xml:112]: I see the concern but I don't have enough experience to know if it's an issue in real life. If it is, we should propose a change to the A2 team. At the moment the CSIP5 description mentions only "OTHER," so I left it unchanged.

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml (hunk 1 above)
Comment (PhillipTommerholt) [reply corpora/csip/metadata/mets-root/CSIP5/testCase.xml]: It says 2.0.3 and not 2.0.4?

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
It says 2.0.3 and not 2.0.4 in line 2?

File: corpora/csip/metadata/mets-root/CSIP5/testCase.xml (hunk 1 above)
Comment (koit) [reply corpora/csip/metadata/mets-root/CSIP5/testCase.xml]: I think I get the issue now: GitHub keeps linking to the version that was valid at the moment of initial review, so the updates made by the later commit 2875094fc9c609c7eed60194ee9f715eec595635 are not there. It seems the "Files changed" tab of the PR always displays the latest state.

**Review by PhillipTommerholt — APPROVED** [review]

**Pull request merged**